ionc/ion_stream_impl.h

```diff
@@ -127,7 +127,6 @@ typedef uint32_t  ION_STREAM_FLAG;
 #define FLAG_CAN_WRITE          0x00200
 #define FLAG_RANDOM_ACCESS      0x00400
 #define FLAG_IS_TTY             0x00800
-#define FD_IS_TTY(fd)           _isatty(fd) /* TODO */
```

Comment:
This is also defined in ion_stream.c but with some platform-specific logic, so this header definition is always overwritten.